### PR TITLE
add clifford recipe

### PR DESCRIPTION
**Pull request by arsenovic**

### CODE REVIEW

File: recipes/clifford/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```
Comment (synapticarbors) [inline recipes/clifford/meta.yaml]: Please remove all of the comments

File: recipes/clifford/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+build:
+  # Uncomment the following line if the package is pure python and the recipe is exactly the same for all platforms.
+  # It is okay if the dependencies are not built for all platforms/versions, although selectors are still not allowed.
+  # See https://conda-forge.org/docs/meta.html#building-noarch-packages for more details.
+  # noarch: python
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  script: python setup.py install --single-version-externally-managed --record record.txt
```
Comment (synapticarbors) [inline recipes/clifford/meta.yaml]: Please use pip 

https://conda-forge.org/docs/meta.html#use-pip

File: recipes/clifford/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+requirements:
+  build:
```
Comment (synapticarbors) [inline recipes/clifford/meta.yaml]: Use the `host` section as in the example recipe:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml

File: recipes/clifford/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    # if your project compiles code (such as a C extension) then add the required compilers as separate entries here.
+    # compilers are named 'c', 'cxx' and 'fortran'.
+    - python
+    - setuptools
+    - numpy
```
Comment (synapticarbors) [inline recipes/clifford/meta.yaml]: Everything except for `python`, `setuptools` and `pip` (once you change the `script` above), should be in the `run` section.

File: recipes/clifford/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+  # List all the packages/modules that `run_test.py` imports.
+  imports:
+    - clifford
+
+about:
+  home: http://clifford.readthedocs.io
+  # Remember to specify the license variants for BSD, Apache, GPL, and LGLP.
+  # Prefer the short version, e.g: GPL-2.0 instead of GNU General Public License version 2.0
+  # See https://opensource.org/licenses/alphabetical
+  license: BSD
```
Comment (synapticarbors) [inline recipes/clifford/meta.yaml]: `license: BSD-3-Clause`

**Comment by arsenovic** [comment]
not sure what this means, any help @synapticarbors ?

File: recipes/clifford/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+  build:
+    - python
+    - pip
+    - setuptools
+    
+  host:
+    - python
+    - pip   
+  run:
+    - numpy
```
[the remark this answers is not in the document]
Comment (arsenovic) [reply recipes/clifford/meta.yaml]: still gives error.

File: recipes/clifford/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+source:
+  url: https://github.com/pygae/{{ name }}/archive/v{{ version }}.zip
+  sha256: ff9ec900c36d28a05307fc259b4465f8c4004e66b6d242f2fb63f4e6d29653b4
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
```
Comment (synapticarbors) [inline recipes/clifford/meta.yaml]: Can you try dropping the `build` section of `requirements`? The `build` section I believe is only necessary when you have compiled code for python packages.

Comment (synapticarbors) [reply recipes/clifford/meta.yaml]: See: https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L35-L44

Comment (arsenovic) [reply recipes/clifford/meta.yaml]: that looks like it, thanks.

**Pull request merged**